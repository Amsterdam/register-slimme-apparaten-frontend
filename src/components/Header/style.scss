@import '~amsterdam-stijl/dist/scss/ams-colorpalette';
@import '~amsterdam-stijl/dist/scss/ams-breakpoints';

.header-component {
  height: 50px;
  display: flex;
  align-items: center;
  box-shadow: 0 2px 0 0 #999999;
  
  .header-link {
    display: flex;
    align-items: center;
    text-decoration: none;

    @media (min-width: 576px) {
      margin-left: calc(8.33333% - 72px);
    }
  }
  
  .map-thumbnail {
    width: 62px;
    height: 30px;
    margin-left: 10px;
    margin-right: 14px;
  }

  .header-title {
    color: $ams-zwart;
  }

}
