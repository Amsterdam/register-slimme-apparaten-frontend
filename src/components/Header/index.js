import React from 'react';
import { NavLink } from 'react-router-dom';

import '../../../public/images/map-thumbnail.png';
import './style.scss';

const Header = () => (
  <div className="header-component no-print">
    <NavLink className="header-link" to="/">
      <img className="map-thumbnail" src={process.env.CITY_HEADER_THUMBNAIL} alt="" />
      <span className="header-title">{process.env.CITY_NAME}</span>
    </NavLink>
  </div>
);

export default Header;
