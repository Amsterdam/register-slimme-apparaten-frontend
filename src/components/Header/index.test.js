import React from 'react';
import { shallow } from 'enzyme';

import Header from './index';

describe('<Header />', () => {
  it('should render correctly', () => {
    const renderedComponent = shallow(
      <Header />
    );
    expect(renderedComponent).toMatchSnapshot();
  });
});
