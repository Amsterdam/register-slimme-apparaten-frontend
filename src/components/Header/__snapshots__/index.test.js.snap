// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should render correctly 1`] = `
<div
  className="header-component no-print"
>
  <NavLink
    activeClassName="active"
    className="header-title"
    to="/"
  >
    Register slimme apparaten
  </NavLink>
</div>
`;
